src/views/conference/2018/index/index.jsx

```diff
@@ -0,0 +1,114 @@
+                        </p>
+                        <table className='conf2018-panel-details'>
+                            <tbody>
+                                <tr className='conf2018-panel-row'>
+                                    <td className='conf2018-panel-row-icon'>
+                                        <img
+                                            className='conf2018-panel-row-icon-image'
+                                            src='/svgs/conference/index/calendar-icon.svg'
+                                            alt='Calendar Icon'
+                                        />
+                                    </td>
+                                    <td><FormattedMessage id='conference-2018.date' /></td>
+                                    <td>
+                                        <FormattedDate
+                                            value={new Date(2018, 6, 26)}
+                                            year='numeric'
+                                            month='long'
+                                            day='2-digit'
+                                        />
+                                        {' - '}
+                                        <FormattedDate
+                                            value={new Date(2018, 6, 28)}
+                                            year='numeric'
+                                            month='long'
+                                            day='2-digit'
+                                        />
+                                        <FormattedMessage id='conference-2018.dateDescMore' />
+                                    </td>
+                                </tr>
+                                <tr className='conf2018-panel-row'>
+                                    <td className='conf2018-panel-row-icon'>
+                                        <img
+                                            className='conf2018-panel-row-icon-image'
+                                            src='/svgs/conference/index/map-icon.svg'
+                                            alt='Map Icon'
+                                        />
+                                    </td>
+                                    <td><FormattedMessage id='conference-2018.location' /></td>
+                                    <td><FormattedMessage id='conference-2018.locationDetails' /></td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </section>
+                    <section className='conf2018-panel'>
+                        <p className='conf2018-panel-desc'>
+                            <FormattedHTMLMessage id='conference-2018.sessionDesc' />
+                        </p>
+                        <p className='conf2018-panel-session'>
+                            <FormattedMessage id='conference-2018.sessionItem1' />
```

Comment:
Remember to switch each item to FormattedHTMLMessage, and wrap each of them in individual paragraphs. For example:
```
<p className='conf2018-panel-session'>
                             <FormattedHTMLMessage id='conference-2018.sessionItem1' />
</p>
```